mlir/utils/jenkins/Jenkinsfile

```diff
@@ -793,5 +817,79 @@ pipeline {
                 }
             }
         }
+        stage ("MIGraphX") {
+            when {
+                beforeAgent true;
+                equals expected: true, actual: params.checkMIGraphX;
```

Comment:
Do these need an anyOf{} ? Not sure. Also, I don't think you defined the check MIGraphX thing

Reply:
I think I defined it here : https://github.com/ROCmSoftwarePlatform/rocMLIR/pull/896/files#diff-16af82a543140fc9970902655eb9815da207d857b849d253ab13c4c0f344aa6aR331 (line 331)

You mean not a AND condition but a OR condition ?

Reply:
Yeah, an OR, so we can set this stage off on PRs if we want to

Reply:
Yea make sense! I ll change that to an OR then.